src/bar.scss

```diff
@@ -28,6 +28,16 @@ $block: #{$fd-namespace}-bar;
     right: $margin-x;
   }
 
+  @mixin padding-left-right($left, $right) {
+    padding-left: $left;
+    padding-right: $right;
+
+    @include fd-rtl() {
+      padding-left: $right;
+      padding-right: $left;
+    }
+  }
```

Comment:
There's already a mixin for this. Please check the `src/mixins/_mixins.scss` file and `fd-set-margins-x` mixin.